data/telegraf_plugins.yml

```diff
@@ -890,7 +890,7 @@ input:
   - name: MySQL
     id: mysql
     description: |
-      The MySQL input plugin gathers the statistics data from MySQL servers.
+      The MySQL input plugin gathers the statistics data from MySQL servers. Input plugin supports MariaDB and Percona.
```

Comment:
What about:

The MySQL input plugin gathers the statistics data from MySQL, MariaDB, and Percona servers.